models/DataObject/ClassDefinition/CustomLayout.php

```diff
@@ -78,9 +78,9 @@ class CustomLayout extends Model\AbstractModel
     protected $layoutDefinitions;
 
     /**
-     * @var int
+     * @var bool
      */
-    protected $default = 0;
+    protected $default = false;
```

Comment:
BC Break? Also the setter cast to int.

Reply:
Hi @blankse, it looks like you are my pair programming mate 🍾 - I just changed the type in 11.x and updated the code here as well. I will discuss the changes (if we change that to bool) internally to find the best possible solution.

BUT your right - that would cause a BC break without changing some other lines 👍

Reply:
@Corepex I would prefer to stay to int in 10.6. You can change it in 11.0 but with Update Notes.

Reply:
@blankse I reverted the changes - we will change the type in 11.x - for now, we just stick to the doctype

Reply:
@Corepex What do you think about allowing int and bool? So we can trigger a deprecation warning if someone put an int to the setter. Also PhpStan would be happy. I would also add this to the Update Notes for 10.6 and 11.0

Reply:
@blankse sounds nice. I`ve implemented the suggested changes and added corresponding upgrade notes ... 

see: https://github.com/pimcore/pimcore/pull/13755/commits/dc38c80fccf3c9e9fefda153d82986a6ff2ee001

Reply:
Should we set the attribute to bool if user give a bool? So the getter also returns int|bool